data-collection/deploy/deploy-data-collection.yaml

```diff
@@ -1,5 +1,5 @@
 AWSTemplateFormatVersion: '2010-09-09'
-Description: CID Data Collection Stack v3.0.7
+Description: CID Data Collection Stack v3.0.8
```

Comment:
Version number is managed by release script automation. Revert to current release number.